packages/sitecore-jss-nextjs/src/index.ts

```diff
@@ -135,3 +140,11 @@ export {
   withPlaceholder,
   withDatasourceCheck,
 } from '@sitecore-jss/sitecore-jss-react';
+
+export { ImageSizeParameters } from '@sitecore-jss/sitecore-jss-react/types/components/Image';
```

Comment:
Could you please add these to the [export of the `sitecore-jss-react` package](https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-react/src/index.ts) and export from root instead of from /types? This makes them available to consumers of the React SDK as well.

Reply:
Good idea, thanks & done 👍🏼